lightning_covid19/model/densenet.py

```diff
@@ -167,12 +195,26 @@ def prepare_data(self):
         else:
             print('Cloning the covid chestxray dataset. It may take a while\n...\n', flush=True)
             run(f'git clone {clone_uri} {chestxray_root}')
+
         transform = transforms.Compose([xrv.datasets.XRayCenterCrop(),
                                         xrv.datasets.XRayResizer(self.hparams.xraysize)])
+
+        # TODO: we could just use K.augmentation.RandomResizedCrop
+        # https://kornia.readthedocs.io/en/latest/augmentation.html#kornia.augmentation.RandomResizedCrop
```

Comment:
Why don't we use the TODO right now?

Reply:
I think it needs some discussion on the strategy about how to use the data.
just to show a couple random cherry picked examples:

![image](https://user-images.githubusercontent.com/5157099/77822161-87248180-70f0-11ea-93db-ffe134b7e8f7.png)

![image](https://user-images.githubusercontent.com/5157099/77822176-a4595000-70f0-11ea-8f8a-b83f4a4b079a.png)

![image](https://user-images.githubusercontent.com/5157099/77822184-b63af300-70f0-11ea-91e3-22c29a8a7776.png)

not sure what would be the best. My guess would the best to analyses the whole image and create some kind of attention to not miss any part.

Another technical note: images are in png, jpg, jpeg as far I can appreciate.

Reply:
> My guess would the best to analyses the whole image and create some kind of attention to not miss any part.

I absolutely agree.
I created an issue describing one possible solution using Bounding Boxes in an issue: #20

Can you please the TODO to the Github issue, from the issue reference relevant part of the code, but please remove the TODO from the code itself. 

Thanks!

Reply:
If you want, there is bot converting todo to issues automatically lol

Reply:
@oplatek will comment there. @Borda would love to see that bot xD

Reply:
https://github.com/marketplace/todo shall I add it?

Reply:
seems not for python :{